### PR TITLE
Making Hextrasenil and Trisyndicotin less stupid.

### PR DESCRIPTION
**Pull request by Kaedwuff**
It really bugged me that someone added magic corporate (dis)loyalty potions to the game along with the trauma system, so I felt a desire to change it so they behaved more like actual medications that just alter your aggression levels.

Several minor changes have been made to these two drugs to make them less like magical Nanotrasen Loyalty Potions/Antagonist Potions.  Now they just generally encourage compliance with authority and disrespect for authority, in a general sense, when working properly.  And the opposite when they fuck up.

They still perform their other functions just fine related to traumas.

https://forums.aurorastation.org/viewtopic.php?f=18&t=10966

### CODE REVIEW

**Comment by BurgerLUA** [comment]
Woah. I'm working on this myself, with feedback from other being taken.

**Comment by Kaedwuff** [comment]
Yeah I know, but... you're pushing the NT loyalty potion idea to it's extreme, instead of fixing it.  

So I made this.

**Comment by BurgerLUA** [comment]
I honestly suggest that you make a seperate medication if you wish to make an all-purpose general drug. You're just fixing a problem that's already being fixed by the codeowner, and your solution is just as stupid as the problem itself.

This PR is a waste of your time, as well as mine. Virtually everything that's changed in this PR will be reset with the introduction of my own fixes.

**Comment by Kaedwuff** [comment]
You're kind of missing the point, man.  I'm not trying to make a new medication, I'm trying to stop these medicines from acting like love/hate potions for Nanotrasen, because that honestly does not make sense to me, in any aspect.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 1]
```diff
@@ -898,17 +898,17 @@
 /datum/reagent/mental/hextrasenil
 	name = "Hextrasenil"
 	id = "hextrasenil"
-	description = "Hextrasenil is a super-strength, fast-metabolizing, expensive antipsychotic medication intended for the use in criminal rehabilitation that treats tourettes, schizophrenia, hallucinations, and loyalty issues. Side effects include undying loyalty to NanoTrasen and respect for authority. Withdrawl effects include undying hatred towards NanoTrasen."
+	description = "Hextrasenil is a super-strength, fast-metabolizing, expensive antipsychotic medication intended for the use in criminal rehabilitation that treats tourettes, schizophrenia, hallucinations, and loyalty issues. Side effects include distractedness and heavy listlessness. Withdrawl effects include increased aggression and paranoia."
```
Comment (OneOneThreeEight) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: Time to be unintentionally annoying with syntax correction suggestions. Remove "heavy" from "heavy listlessness." Emphasis doesn't seem needed.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 2]
```diff
@@ -898,17 +898,17 @@
...
 	reagent_state = LIQUID
 	color = "#888888"
 	metabolism = 0.04 //Not meant to last a long time.
 	data = 0
-	taste_description = "paper"
-	goodmessage = list("You feel loyal to NanoTrasen. Please take your pills.","You feel the need to contribute to cause of NanoTrasen. Please take your pills.","You wouldn't think about hurting NanoTrasen at all. Please take your pills.","You do not feel the need to express emotion. Please take your pills.","You feel that NanoTrasen has your best interests at heart. Please take your pills.","You respect the Chain of Command. Please take your pills.")
+	taste_description = "warm milk"
+	goodmessage = list("You feel lazy and detatched...","You start to tune out, and the world around you quiets to a pleasing hum..","You wouldn't think about hurting anyone...","You feel calm and emotionless...","You feel that everyone around you has your best interest at heart...","You feel increased respect for authority..")
```
Comment (OneOneThreeEight) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: Detached.*
best interests at heart*

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 3]
```diff
@@ -898,17 +898,17 @@
...
 	badmessage = list("You start to think if you need these pills...","Do I need these pills?","Should I be taking pills anymore?")
-	worstmessage = list("You start to realise that the system is corrupt...","NanoTrasen is corrupt...")
+	worstmessage = list("You start to realise that they're out to get you...","They're call corrupt, and you can trust no one...")
```
Comment (OneOneThreeEight) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: They're all corrupt*

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 4]
```diff
@@ -920,15 +920,15 @@
...
 	reagent_state = LIQUID
 	color = "#888888"
 	metabolism = 0.03
 	data = 0
-	taste_description = "freedom"
-	goodmessage = list("You distrust Nanotrasen and their people.","You feel woke.","You have urges to speak out against NanoTrasen.","You feel the need to complain about NanoTrasen on the web.","You feel like things should be better.")
-	badmessage = list() //Actual Freedom.
-	worstmessage = list() //Actual Freedom.
+	taste_description = "bitterness"
+	goodmessage = list("You feel the desire to speak your mind...","You feel awakened to the truth - your coworkers are your enemies...","You have urges to speak out against your decietful coworkers...","You feel the need to complain about how much you dislike your superiors...","You feel like things should be better...")
```
Comment (OneOneThreeEight) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: Deceitful*

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 5]
```diff
@@ -920,15 +920,15 @@
...
+	badmessage = list("What did they put in you...?","Your mind feels hazy, and the faces around you waver.") 
+	worstmessage = list("Why should you have to listen to what anyone tells you..?","You have to get your coworkers before they get you...",'KILL THEM ALL BEFORE THEY KILL YOU.')
```
Comment (OneOneThreeEight) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: Is turning people berserk a good idea? Hallucinated ramblings might be better, and people can choose if they want to get violent or turn into vegetables.

**Comment by Kaedwuff** [comment]
It's perfectly possible for you to keep the rest of your code changes and still remove your changes to Hextrasenil and Trisyndicotin if people prefer my take on them, including intentionally changing their name.

What I've done here is an alternate route, and in no way invalidates your entire PR, which I can see even now in the comments and forum thread is meeting with some resistance.  These are pull _requests_, not pull obligations.  People are under no obligation to push your code changes through if they wish to do something else.

The same goes for me.

**Comment by OneOneThreeEight** [comment]
There's nothing explicitly wrong with contributors competing with different solutions to a PR, especially if there is a heavy disagreement with the initial implementation.

Rocriodide - An aggression inhibitor. The former pro-loyalty thing.
Qospivanadate - An aggression stimulant. The former pro-antag thing.

**Comment by Kaedwuff** [comment]
There, name changes implemented.

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 6]
```diff
@@ -322,30 +322,30 @@
 		new /obj/item/weapon/reagent_containers/pill/escitalopram( src )
 		new /obj/item/weapon/reagent_containers/pill/escitalopram( src )
 		
-/obj/item/weapon/storage/pill_bottle/trisyndicotin
-	name = "bottle of Trisyndicotin pills"
-	desc = "Used to free one's mind from the oppression of NanoTrasen."
+/obj/item/weapon/storage/pill_bottle/Qospivanadate
```
Comment (LordFowl) [inline code/game/objects/items/weapons/storage/firstaid.dm]: This should be lowercase.

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 7]
```diff
@@ -322,30 +322,30 @@
...
+		new /obj/item/weapon/reagent_containers/pill/Qospivanadate( src )
+		new /obj/item/weapon/reagent_containers/pill/Qospivanadate( src )
+		new /obj/item/weapon/reagent_containers/pill/Qospivanadate( src )
+		new /obj/item/weapon/reagent_containers/pill/Qospivanadate( src )
+		new /obj/item/weapon/reagent_containers/pill/Qospivanadate( src )
+		new /obj/item/weapon/reagent_containers/pill/Qospivanadate( src )
 		
-/obj/item/weapon/storage/pill_bottle/hextrasenil
-	name = "bottle of Hextrasenil pills"
+/obj/item/weapon/storage/pill_bottle/Rocriodide
```
Comment (LordFowl) [inline code/game/objects/items/weapons/storage/firstaid.dm]: This should be lowercase.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 8]
```diff
@@ -895,20 +895,20 @@
 		/datum/reagent/mental/escitalopram = 20
 	)
 
-/datum/reagent/mental/hextrasenil
-	name = "Hextrasenil"
-	id = "hextrasenil"
-	description = "Hextrasenil is a super-strength, fast-metabolizing, expensive antipsychotic medication intended for the use in criminal rehabilitation that treats tourettes, schizophrenia, hallucinations, and loyalty issues. Side effects include undying loyalty to NanoTrasen and respect for authority. Withdrawl effects include undying hatred towards NanoTrasen."
+/datum/reagent/mental/Rocriodide
```
Comment (LordFowl) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: Lowercase.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 9]
```diff
@@ -895,20 +895,20 @@
...
+	name = "Rocriodide"
+	id = "rocriodide"
+	description = "Rocriodide is a super-strength, fast-metabolizing, expensive antipsychotic medication intended for the use in criminal rehabilitation that treats tourettes, schizophrenia, hallucinations, and loyalty issues. Side effects include distractedness and listlessness. Withdrawl effects include increased aggression and paranoia."
```
Comment (LordFowl) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: Typo. Withdrawl should be Withdrawal.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 10]
```diff
@@ -917,18 +917,18 @@
 	)
 	withdrawal_traumas = list()
 
-/datum/reagent/mental/trisyndicotin
-	name = "Trisyndicotin"
-	id = "trisyndicotin"
-	description = "Trisyndicotin is a super-strength, expensive antipsychotic medication intended for the use in interigation. Side effects include undying hatred to NanoTrasen and disrespect for authority."
+/datum/reagent/mental/Qospivanadate
```
Comment (LordFowl) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: Lowercase.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 11]
```diff
@@ -588,17 +588,17 @@
 	required_reagents = list("mindbreaker" = 1, "space_drugs" = 1, "silicon" = 1)
 	result_amount = 3
 
-/datum/chemical_reaction/hextrasenil
-	name = "Hextrasenil"
-	id = "hextrasenil"
-	result = "hextrasenil"
+/datum/chemical_reaction/Rocriodide
```
Comment (LordFowl) [inline code/modules/reagents/Chemistry-Recipes.dm]: Lowercase.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 12]
```diff
@@ -588,17 +588,17 @@
...
+	id = "rocriodide"
+	result = "Rocriodide"
 	required_reagents = list("truthserum" = 1, "risperidone" = 1, "mindbreaker" = 1)
 	result_amount = 3
 
-/datum/chemical_reaction/trisyndicotin
-	name = "Trisyndicotin"
-	id = "trisyndicotin"
-	result = "trisyndicotin"
+/datum/chemical_reaction/Qospivanadate
```
Comment (LordFowl) [inline code/modules/reagents/Chemistry-Recipes.dm]: Lowercase.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 13]
```diff
@@ -171,21 +171,21 @@
 		. = ..()
 		reagents.add_reagent("escitalopram", 15)
 		
-/obj/item/weapon/reagent_containers/pill/hextrasenil
-	name = "Hextrasenil pill"
+/obj/item/weapon/reagent_containers/pill/Rocriodide
```
Comment (LordFowl) [inline code/modules/reagents/reagent_containers/pill.dm]: Lowercase.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 14]
```diff
@@ -171,21 +171,21 @@
...
 	desc = "Don't forget to take your pills."
 	icon_state = "pill6"
 	Initialize()
 		. = ..()
-		reagents.add_reagent("hextrasenil", 30)
+		reagents.add_reagent("Rocriodide", 30)
 		
-/obj/item/weapon/reagent_containers/pill/trisyndicotin
-	name = "Trisyndicotin pill"
+/obj/item/weapon/reagent_containers/pill/Qospivanadate
```
Comment (LordFowl) [inline code/modules/reagents/reagent_containers/pill.dm]: Lowercase.

**Comment by LordFowl** [comment]
Closed by author's request.

**Pull request closed without merging**